src/gui/cookiesdialog.h

```diff
@@ -1,6 +1,6 @@
 /*
- * Bittorrent Client using Qt4 and libtorrent.
- * Copyright (C) 2010  Christophe Dumez
```

Comment:
git generated wrong diff here! I not edited old cookiesdlg.h. I just deleted it and created new cookiesdialog.h not based on it.